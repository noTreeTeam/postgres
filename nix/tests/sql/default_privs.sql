-- this tests the outcome of doing ALTER DEFAULT PRIVILEGES..
select defaclrole::regrole, defaclnamespace::regnamespace, defaclobjtype from pg_default_acl where defaclnamespace = 'public'::regnamespace::oid order by defaclrole::regrole, defaclobjtype;
