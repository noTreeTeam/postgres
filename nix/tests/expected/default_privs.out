-- this tests the outcome of doing ALTER DEFAULT PRIVILEGES..
select defaclrole::regrole, defaclnamespace::regnamespace, defaclobjtype from pg_default_acl where defaclnamespace = 'public'::regnamespace::oid order by defaclrole::regrole, defaclobjtype;
   defaclrole   | defaclnamespace | defaclobjtype 
----------------+-----------------+---------------
 supabase_admin | public          | S
 supabase_admin | public          | f
 supabase_admin | public          | r
 postgres       | public          | S
 postgres       | public          | f
 postgres       | public          | r
(6 rows)

